frontend/src/components/Trial/Trial.jsx

```diff
@@ -23,7 +23,7 @@ const Trial = ({
 }) => {
     // Main component state
     const [formActive, setFormActive] = useState(!config.listen_first);
-    const [preloadReady, setPreloadReady] = useState(!(playback?.ready_time));
+    const [preloadReady, setPreloadReady] = useState(false);
```

Comment:
It's actually this part which is problematic, as we don't have a `setPreloadReady` command in Trials without Playback.